### DISCUSSION TITLE
Post pics of your setup for the Readme

### DISCUSSION THREAD

**Discussion by agronick** [agronick/aa-torque#24] [General]
Some of the images in the readme don't look great with the text on multiple lines.  If you take the time to make everything look good with icons post a picture of your setup here so it can be added to the readme.

**Comment by Vince56190** [comment]
2012 Lancia Thema , custom Android Auto (EngCustom unit ) Pixel 6A , Android 14
![Capture d'écran 2023-09-29 175200](https://github.com/agronick/aa-torque/assets/147246549/63414b98-c333-4760-8e71-772ba823d65b)

**Comment by Vince56190** [comment]
A youtube video if you want : https://youtu.be/ijNiJ47pz5A
Car : Lancia Thema 2012 , V6 , 3Liter Turbo Diesel.

**Comment by mkwatchface** [comment]
Perhaps you can use these pictures:

![20231026_080855](https://github.com/agronick/aa-torque/assets/62142158/d56ba184-b05b-461d-9890-d33575c6e8b7)


![20231026_080913](https://github.com/agronick/aa-torque/assets/62142158/043ae547-befb-41f0-970f-bdad5d32d44e)

The Car is a Skoda Scala Monte Carlo. This is a view from further away:

![20231026_081005](https://github.com/agronick/aa-torque/assets/62142158/0d8ded34-9528-48db-a285-e81e4b849717)

![20231026_081021](https://github.com/agronick/aa-torque/assets/62142158/48db439b-9840-4d6e-afec-afbc2cd53a92)

**Reply by agronick** [reply]
Those look really nice. Those are the highest quality ones we have so far. I made it first.

**Reply by mkwatchface** [reply]
Thank you! i am glad to support!

**Comment by Vince56190** [comment]
https://zupimages.net/up/23/43/0sle.jpg
![20231028_182017](https://github.com/agronick/aa-torque/assets/147246549/61c2176f-a6b0-4442-afef-c52988e1607b)

**Comment by aross6** [comment]
2022 Kia Seltos SX 1.6L Turbo GDI 
Android Auto
Torque Pro
AA Torque
Gauge setup: 
![PXL_20231029_162650359~2](https://github.com/agronick/aa-torque/assets/24236718/211fe852-c41f-4015-8455-49db128dc53d)

**Comment by ghynett** [comment]
![20231006_063216](https://github.com/agronick/aa-torque/assets/60058899/4edcc554-23d1-4e4d-8796-bc314bc45943)
Citroën C3
